doc/htmldoc/clean_source_dirs.py

```diff
@@ -1,35 +1,35 @@
-# -*- coding: utf-8 -*-
-#
-# clean_source_dirs.py
-#
-# This file is part of NEST.
-#
-# Copyright (C) 2004 The NEST Initiative
-#
-# NEST is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 2 of the License, or
-# (at your option) any later version.
-#
-# NEST is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
-import os
-import pathlib
-import shutil
-from glob import glob
-
-for dir_ in ("auto_examples", "models"):
-    for file_ in glob(str(pathlib.Path(__file__).parent / dir_ / "*")):
-        if not any(file_.endswith(f) for f in (".gitignore", "models-main.rst", "models-toc.rst")):
-            try:
-                try:
-                    os.unlink(file_)
-                except OSError:
-                    shutil.rmtree(file_)
-            except Exception:
-                print(f"Couldn't remove '{file_}'")
+# -*- coding: utf-8 -*-
+#
+# clean_source_dirs.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+import os
+import pathlib
+import shutil
+from glob import glob
+
+for dir_ in ("auto_examples", "models"):
+    for file_ in glob(str(pathlib.Path(__file__).parent / dir_ / "*")):
+        if not any(file_.endswith(f) for f in (".gitignore", "index.rst")):
+            try:
+                try:
+                    os.unlink(file_)
+                except OSError:
+                    shutil.rmtree(file_)
+            except Exception:
+                print(f"Couldn't remove '{file_}'")
```

Comment:
```suggestion
from pathlib import Path
import shutil

base = Path(__file__).parent      # base directory of this file
for dirname in ("auto_examples", "models"):
    for entry in (base / dirname).glob():
        if entry.name in (".gitignore", "index.rst"):    # keep these files
            continue
        try:       # delete everything else
            if entry.isdir():
                shutil.rmtree(str(entry))
            else:
                entry.unlink()
        except Exception as exc:
            print(f"Couldn't remove '{entry}': {exc}")
```
Not sure this always reacts sanely, as the script will also delete itself...

Reply:
I wouldn't make these change here in this PR as it's not directly related; the change that is needed is only the name of file. 

The only reason the entire file appears to be 'new' is because there was Windows newlines  that were removed to all lines.